pkg/test/framework/components/istio/config.go

```diff
@@ -100,9 +106,15 @@ type Config struct {
 	// UndeployTimeout the timeout for undeploying Istio.
 	UndeployTimeout time.Duration
 
-	// The IstioOperator spec file to be used for defaults
+	// The IstioOperator spec file to be used for Control plane cluster by default
 	IOPFile string
 
+	// The IstioOperator spec file to be used for Config cluster by default
+	ConfigIOPFile string
```

Comment:
Could we make it `ConfigClusterIOPFile` and `RemoteClusterIOPFile` to be more clear?

Reply:
fixed